go/vt/sqlparser/normalizer_test.go

```diff
@@ -228,6 +242,13 @@ func TestNormalize(t *testing.T) {
 		outbv: map[string]*querypb.BindVariable{
 			"bv1": sqltypes.TestBindVariable([]any{1, "2"}),
 		},
+	}, {
+		// EXPLAIN queries
```

Comment:
Did this case already work correctly and is this just to add test coverage?

Reply:
Yep